### ISSUE TITLE
incorrectly tagged lint "Use FALSE instead of the symbol F"

### ISSUE THREAD

**Issue by IndrajeetPatil** [jimhester/lintr#657]
Since a couple of days, I have started to get a new lint in my CI checks (https://github.com/IndrajeetPatil/statsExpressions/commit/cc00f65c4c6640c51cea165f6a557f13c097db85#commitcomment-44804914).

But I think this is unintentional, since here `F` refers to `F`-statistic

```r
R/helpers_anova.R:134:34: style: Use FALSE instead of the symbol F.

​        statistic = ez_df$ANOVA$F[2],

                                ~^
```

**Comment by AshesITR** [comment]
Thanks for reporting.
In the mean time, you can use

`ez_df$ANOVA[["F"]][2]` to work around the issue.

**Comment by AshesITR** [comment]
This should only concern dollar-expressions (where SYMBOLs aren't evaluated), argument names, assignments and NSE (which would be difficult to support IINM)

No T_and_F lints should fire for
```
x <- list(
  T = 42L,
  F = 21L
)

x$F <- 42L

T <- "foo"
```
